### PR TITLE
Support matplotlib 3.9.0

### PR DESCRIPTION
**Pull request by xiki-tempula**
Fix https://github.com/alchemistry/flamel/issues/28

### CODE REVIEW

**Comment by xiki-tempula** [comment]
@orbeckst Do you mind do a quick review? I will release this version after the PR is merged as this is a critical bug fix.

**Review by orbeckst — APPROVED** [review]
lgtm, thanks!

File: CHANGES [hunk 1]
```diff
@@ -21,6 +21,9 @@ Changes:
   - alchemlyb adopts SPEC 0 (replaces NEP 29)
     https://scientific-python.org/specs/spec-0000/
 
+Fixes
+  - Support matplotlib 3.9.0 (issue #28, PR#319).
```
Comment (orbeckst) [inline CHANGES:25]:
```suggestion
  - Support matplotlib 3.9.0 (issue alchemistry/flamel#28, PR#319).
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
What is the minimum version of matplotlib that is working with this change? Update pyproject.toml, please.

We will also need this information for the conda-forge package.

**Comment by sidneypaulymer** [comment]
LGTM! I reinstalled the bugfix branch into a fresh environment, and `flamel` now completes without an error. Thanks! My only concern has already been raised: what is the minimum version of `matplotlib` that will support this change?

**Comment by xiki-tempula** [comment]
minimum version of matplotlib is 3.7.

**Review by orbeckst — APPROVED** [review]
All good, please squash-merge when you want.

**Pull request merged**